keyboards/draytronics/scarlet/scarlet.h

```diff
@@ -0,0 +1,35 @@
+/*Copyright 2020 Blake Drayson / Draytronics
+
+Contact info@draytronics.co.uk
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_numpad_5x4( \
+  K00, K01, K02, K03, \
+  K10, K11, K12, K13, \
+  K20, K21, K22, \
+  K30, K31, K32, K33,\
+  K40,      K42 \
```

Comment:
@zvecr's request was done to enable this board to use the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts). Using that feature means that every keyboard that supports a given layout – in your case `numpad_5x4` – has to assign their keycodes in the same sequence. For `LAYOUT_numpad_5x4` that means assigning `KC_PPLS` after `KC_P6`, and `KC_PENT` after `KC_PDOT`.

The change to `keymap.c` ties into this. You have to make both sets of changes, or neither (and thus not use the Community Layouts feature).

```suggestion
  K10, K11, K12,      \
  K20, K21, K22, K13, \
  K30, K31, K32,      \
  K40,      K42, K33  \
```

Also, as an aside, the top section of the layout macro doesn't have to match the electrical matrix. The top section controls the sequence of keycode assignment, and is meant to resemble the layout of the assembled board, but it's the bottom section (Lines 30-34) that designates how the board is wired electrically.

Reply:
Thank you for this, my apologies for a lack of understanding in this area. It makes perfect sense now, I appreciate you taking the time to explain this, it is something that I should have understood better! I shall make the required changes.